plugins/toolkit/jetbrains-core/resources/META-INF/ext-java.xml

```diff
@@ -48,13 +48,6 @@
     </extensions>
     <actions>
         <group id="aws.toolkit.codemodernizer.toolbar">
-            <action
-                id="codemodernizer.toolbar.startmodernizer"
-                class="software.aws.toolkits.jetbrains.services.codemodernizer.actions.CodeModernizerStartModernizerAction"/>
-            <action
```

Comment:
Question: Why are we getting rid of the "stop" action? The only stop button would be in the chat and until we get rid of the bottom panel progress window, I don't think we should remove the stop action from there.

Reply:
Added it back, you're right that we should keep it